test/functional/p2p_invalid_messages.py

```diff
@@ -6,6 +6,7 @@
 import struct
 import sys
 
+
```

Comment:
19304: should be marked partial: missing changes in `test_large_inv` which we don't have yet

Reply:
done